crates/libcontainer/src/container/init_builder.rs

```diff
@@ -90,10 +90,15 @@ impl<'a> InitContainerBuilder<'a> {
             notify_path,
             container: Some(container.clone()),
             preserve_fds: self.base.preserve_fds,
-            detached: false, // TODO this should be set properly based on how the command is given
+            // TODO: This should be set properly based on how the command is
+            // given. For now, set the detached to true because this is what
+            // `youki create` defaults to.
+            detached: true,
```

Comment:
Please create the issue about this comment.

Reply:
I will do this. I was planning to fix this right after since this will be required to enable foreground mode. I want this PR to make all the detached mode situation work and `youki create` defaults to detached but `youki run` has a flag to switch between foreground and background mode.

Reply:
Actually https://github.com/containers/youki/issues/317 and https://github.com/containers/youki/issues/628 covers the situation fairly well.